minutes/minutes-2015-03-12.md

```diff
@@ -0,0 +1,368 @@
+
+* In Person Meeting Timing Doodle Poll
+
+    * Assigned: Greg
+
+    * We need to have a doodle poll to determine the right time to have a meeting (in Cold Spring Harbor?)
+
+* Put a Clean Version of the Finance Spreadsheet in the repository
+
+    * Assigned: Jason
+
+    * Done (differently): Created a google spreadsheet, which has been made available to GVW, KH and JW.
+
+    * Financial Report
+
+    * Jason
+
+# ## Updates from Subcommittees [ 20 min, until 0:35 ]
+
+Updates only necessary if applicable. It's OK to say "no news from X Subcommittee".
+
+* Fees
+
+    * Liaison: Raniere Silva
+
+    * Update: Met with Aleksandra, Karin and Camille. 
+
+* Finance
+
+    * Liaison: Jason Williams
+
+    * Update: 
+
+        1. Draft charge has been made available to committee.
+
+        2. Mailing list created for this subcommittee: finance@lists.software-carpentry.org 
+
+* Partner Relations / Development
+
+    * Liaison: Jason Williams
+
+    * Update: 
+
+        3. We have met and are working on the draft charge, to be approved by the subcommittee by 3/20. 
+
+        4. Ideas are being collected by Trello for final prioritization on 3/20.
+
+        5. Tom Wright has agreed to take on one of the subcommittee leadership roles, we are still working on having two ‘leaders’; Chair and Secretary 
+
+        6. Mailing list created for this subcommittee: development@lists.software-carpentry.org 
+
+* Assessment
+
+    * Liaison: Jason Williams
+
+    * Update: 
+
+        7. Leadership roles have been determined for this committee, Daniel Chen as Chair, Jeramia Ory as Secretary (through July 2015) 
+
+        8. Daniel and Jeramia will now meet with Jason and Jory S. (+ others?) to prioritize collected ideas and reconvene the whole committee. Has created draft charge.
+
+        9. Mailing list revived: assessment@lists.software-carpentry.org
+
+	
+
+* Mentorship
+
+    * Liaison: Raniere Silva
+
+    * Update: Sheldon and Rayna has run an instructor debrief meeting.
+
+* Lesson Organization and Development
+
+    * Liaison: Matt Davis
+
+* Communication
+
+    * Liaison: Adina Chuang Howe
+
+    * Update:
+
+        10. Assigned:  brochure, badge, blogposts, description of workshops, press release template, summarizing steering committee activities, grant text
+
+        11. Open issues with no assignments:  twitter, annual report
+
+# ## Translations [ 10 min, until 0:45]
+
+* Lead: Greg
+
+* How do we deal with translated versions of the curriculum?
+
+# ## Corporate SWC Workshops [ 10 min, until 0:55 ]
+
+* Lead: Greg
+
+* There was a discussion about Monsanto and corporations in general having workshops.
+
+* Update from Greg on this topic
+
+* Anything we need to decide or deal with as SCF SC?
+
+# ## Issues in the board repo [ 10 min, until 1:10 ]
+
+Trevor has made some suggestions as some combination of issues and pull
+
+requests. I would like to vote on the ones that require board approval.
```

Comment:
Probably drop thte blank line in the middle of this paragraph.  I can submit a fixup PR (since this one is already merged) if that's easier than putting comments here.